src/components/UploadZzal/ImageUpload.tsx

```diff
@@ -0,0 +1,115 @@
+import { Fragment, useRef, useState, DragEvent, ChangeEvent } from "react";
+import { Upload } from "lucide-react";
+import { cn } from "@/utils/tailwind";
+import ZzalCard from "@/components/common/ZzalCard";
+
+type DivDragEvent = DragEvent<HTMLDivElement>;
+interface Props {
+  onChange: (file: File | null) => void;
+}
+
+const ImageUpload = ({ onChange }: Props) => {
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+  const [dragging, setDragging] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const handleClickDeleteButton = () => {
+    onChange(null);
+    setPreviewUrl(null);
+  };
+
+  const handleFileChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const { files } = event.target;
+    if (files) {
+      const [changedFile] = files;
+      onChange(changedFile);
+      if (changedFile) {
+        setPreviewUrl(URL.createObjectURL(changedFile));
+      }
```

Comment:
```suggestion
      if (changedFile) {
        setPreviewUrl(URL.createObjectURL(changedFile));
        onChange(changedFile);
      }
```
이렇게 작성해주면 `changedFile`이 존재할 때 파일 처리를 하도록 해줄 수 있을 것 같은데 어떻게 생각하시나요?

추가로 유효하지 않은 url일 경우의 처리를 해주면 좋을 것 같다는 생각을 했는데 api연결할 때 처리해줘도 좋을 것 같습니다!